src/Accordion/Accordion.tsx

```diff
@@ -85,7 +92,7 @@ const Accordion: React.FunctionComponent<AccordionProps> = (props: AccordionProp
                         role="button"
                     >
                         <div
-                            className={"header-wrapper" + (item.subHeaderText ? " with-sub-header" : "")}
+                            className={"header-wrapper" + (item.subHeaderText ? " with-sub-header" : "") + (props.alternative && !props.customIcon ? " alternate-header" : "")}
```

Comment:
Id doable, move this to a variable too.

Reply:
removed the newly added condition as it is not necessary to set default icon for new theme